project/SparkBuild.scala

```diff
@@ -910,7 +910,7 @@ object Unidoc {
       .map(_.filterNot(_.getCanonicalPath.contains("org/apache/spark/sql/catalyst")))
       .map(_.filterNot(_.getCanonicalPath.contains("org/apache/spark/sql/execution")))
       .map(_.filterNot(_.getCanonicalPath.contains("org/apache/spark/sql/internal")))
-      .map(_.filterNot(_.getCanonicalPath.contains("org/apache/spark/sql/hive/test")))
+      .map(_.filterNot(_.getCanonicalPath.contains("org/apache/spark/sql/hive")))
```

Comment:
@xuanyuanking, can we avoid doing this here? Making private sounds correct at this moment. Hive module at least has one API exposed `HiveSessionStateBuilder`.

Reply:
Also, we should document that this is a private module at https://github.com/apache/spark/blob/master/sql/hive/src/main/scala/org/apache/spark/sql/hive/package.scala. We could do it separately in another PR.

Reply:
I don't think `HiveSessionStateBuilder` is a public API. It's base class, `BaseSessionStateBuilder`, is in `org.apache.spark.sql.internal`.

Reply:
Then we should remove `@Unstable`. Can we do this in a separate PR?

Reply:
yea we should do it in a separated PR.

Reply:
Copy that, will do it in a separated PR.

Reply:
Done in #31321